packages/react-scripts/template/README.md

```diff
@@ -409,6 +409,50 @@ In development, expressing dependencies this way allows your styles to be reload
 
 If you are concerned about using Webpack-specific semantics, you can put all your CSS right into `src/index.css`. It would still be imported from `src/index.js`, but you could always remove that import if you later migrate to a different build tool.
 
+## Adding a CSS Modules based stylesheet.
```

Comment:
If it's not auto-generated, you might want to add an entry to the table of contents at the top of the file.